app/policies/receipt_policy.rb

```diff
@@ -3,13 +3,38 @@
 class ReceiptPolicy < ApplicationPolicy
   def destroy?
     user&.admin? ||
-      (record&.receiptable&.event&.users&.include?(user) && unlocked?) ||
+      (record&.receiptable&.event &&
+        OrganizerPosition.role_at_least?(user, record.receiptable.event, :member) &&
+        unlocked?
+      ) ||
       # Checking if receiptable is nil prevents unauthorized
       # deletion when user no longer has access to an org
       (record&.receiptable.nil? && record&.user == user) ||
       (record&.receiptable.instance_of?(Reimbursement::Expense) && record&.user == user && unlocked?)
   end
 
+  def destroy?  
```

Comment:
Looks like a merge gone wrong here (likely why CI is failing). We have two `destroy?` method definitions.